src/js/configHolder.js

```diff
@@ -0,0 +1,153 @@
+"use strict";
+var fluid = require("infusion");
+var process = require("process");
+
+// TODO: Revisit once we can use a package-relative path.
+// https://github.com/fluid-project/markdownlint-config-fluid/pull/2
```

Comment:
Looks like this got merged now

Reply:
Great, I updated to use `fluid.require` as an expander instead.